src/get-bounding-client-rect.js

```diff
@@ -48,6 +47,10 @@ function getBoundingClientRect() {
  * @return {boolean}
  */
 function shouldInstall(win) {
+  if (!win.document) {
+    return false;
+  }
```

Comment:
Looks like this will no longer be called from src/polyfills.js but changes are probably worth doing anyways.

Reply:
I realized this as well and came to the same conclusion. Still, I'm happy to back these out since they're not really necessary. Just let me know.